lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/sys.rb

```diff
@@ -1224,8 +1224,10 @@ def cmd_drop_token(*args)
   #
   def cmd_sysinfo(*args)
     info = client.sys.config.sysinfo(refresh: true)
-    client.update_session_info
-
+    begin
+      client.update_session_info
+    rescue
```

Comment:
```suggestion
    rescue Exception => e
    elog e
    print_error(e.to_s)
```
This should get you the error.